<?xml version="1.0"?>
<launch>
    <node name="planner" pkg="elvez_pkg" type="planner.py" output="screen">
    </node>
</launch>  
